app/src/main/java/uk/gov/onelogin/developer/tabs/app/AppTabScreenViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.onelogin.developer.tabs.app
+
+import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.json.Json
+import uk.gov.onelogin.appinfo.apicall.domain.model.AppInfoData
+import uk.gov.onelogin.appinfo.source.domain.model.AppInfoLocalState
+import uk.gov.onelogin.appinfo.source.domain.source.AppInfoLocalSource
+
+@HiltViewModel
+class AppTabScreenViewModel @Inject constructor(
+    private val appInfoLocalSource: AppInfoLocalSource
```

Comment:
Isn't AppInfoService supposed to be used as the single source of truth (repository) for the AppInfo call and data?

Reply:
correct, in the case of the dev screen to show this 'saved' data, we are just hooking into it